client/post-editor/post-editor.jsx

```diff
@@ -352,6 +354,7 @@ const PostEditor = React.createClass( {
 										/>
 									: null
 								}
+								<QueryPreferences/>
```

Comment:
It shouldn't make any effective difference, but might be nicer to see this rendered as an immediate child of the root returned element, to quickly grok which query components are in use by the component.